packages/rps/src/redux/metamask/actions.ts

```diff
@@ -2,7 +2,6 @@ export const METAMASK_ERROR = 'METAMASK.ERROR';
 export const METAMASK_SUCCESS = 'METAMASK.SUCCESS';
 export const enum MetamaskErrorType {
   WrongNetwork = 'WrongNetwork',
-  NoWeb3 = 'NoWeb3',
```

Comment:
It seems to me this is catching the case of MetaMask not even being installed, and so we probably still need some additional error type for that.

Reply:
Good point. Will fix.

Reply:
See 11c42d1

Reply:
![Screenshot 2019-12-16 at 15 07 14](https://user-images.githubusercontent.com/1833419/70917980-3774eb00-2016-11ea-9a30-8fa05534b4e6.png)